eva/third_party/huggingface/create.py

```diff
@@ -122,7 +124,7 @@ def infer_output_name_and_type(**pipeline_args):
 
     # Get a dictionary of output names and types from the output
     output_types = {}
-    if isinstance(model_output, list):
+    if isinstance(model_output, list) and model_output != []:
```

Comment:
Do we need this additional check? I see you changed the default example to include entities, so model_output would never be equal to `[]`. You can do away with this check probably.

Reply:
made this changes and pushed